simulator/src/sequential/TflipFlop.js

```
@@ -73,7 +73,9 @@ export default class TflipFlop extends CircuitElement {
      * We flip the bits to find qInvOutput
      */
     resolve() {
-        if (this.reset.value == 1) {
+        if (this.preset.value == 1) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)